packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -65,13 +67,17 @@ class StdoutLogger extends Logger {
   }
 
   @override
-  void printStatus(String message, { bool emphasis: false, bool newline: true, String ansiAlternative }) {
+  void printStatus(
+    String message,
+    { bool emphasis: false, bool newline: true, String ansiAlternative, int indent }) {
```

Comment:
ditto

the rule is just to make sure the whitespace at the open bracket matches the whitespace at the close bracket.